pkg/controllers/chart/chart.go

```diff
@@ -355,6 +357,11 @@ func job(chart *v1.HelmChart) (*batch.Job, *corev1.ConfigMap, *corev1.ConfigMap)
 		targetNamespace = chart.Spec.TargetNamespace
 	}
 
+	chartName := chart.Spec.Chart
+	if chart.Spec.Repo != "" {
+		chartName = chart.Name + "/" + chart.Spec.Chart
```

Comment:
This seems a bit confusing, but it works with how the klipper-helm script installs the chart.
1. the chart repo is added using `chart.Name` as the repo name
2. the chart needs to be referenced as `repo/chart` in order for credentials specified when the repo is added to be used when installing or updating the chart. If the chart is referenced without a `repo/` prefix, but the bare repo URL passed in the `--repo` flag, the credentials will not be used.